src/mam4xx/convproc.hpp

```diff
@@ -867,21 +1216,21 @@ void initialize_dcondt(const bool doconvproc_extd[ConvProc::pcnst_extd],
    in :: iflux_method             ! 1=as in convtran (deep), 2=uwsh
    in :: ktop                     ! top level index
    in :: kbot                     ! bottom level index
-   in :: dpdry_i(pver)            ! dp [mb]
-   in :: fa_u(pver)               ! fractional area of the updraft [fraction]
-   in :: mu_i(pverp)              ! mu at current i (note pverp dimension, see ma_convproc_tend) [mb/s]
-   in :: md_i(pverp)              ! md at current i (note pverp dimension) [mb/s]
-   in :: chat(pverp,pcnst_extd)   ! mix ratio in env at interfaces [kg/kg]
-   in :: gath(pver,pcnst_extd)   ! gathered tracer array [kg/kg]
-   in :: conu(pverp,pcnst_extd)   ! mix ratio in updraft at interfaces [kg/kg]
-   in :: cond(pverp,pcnst_extd)   ! mix ratio in downdraft at interfaces [kg/kg]
-   in :: dconudt_activa(pverp,pcnst_extd) ! d(conu)/dt by activation [kg/kg/s]
-   in :: dconudt_wetdep(pverp,pcnst_extd) ! d(conu)/dt by wet removal[kg/kg/s]
-   in :: dudp(pver)           ! du(i,k)*dp(i,k) at current i [mb/s]
-   in :: dddp(pver)           ! dd(i,k)*dp(i,k) at current i [mb/s]
-   in :: eudp(pver)           ! eu(i,k)*dp(i,k) at current i [mb/s]
-   in :: eddp(pver)           ! ed(i,k)*dp(i,k) at current i [mb/s]
-   out :: dcondt(pver,pcnst_extd)  ! grid-average TMR tendency for current column  [kg/kg/s]
+   in :: dpdry(nlev)              ! dp [mb]
```

Comment:
Not a big deal, but maybe using square brackets here as you've done elsewhere would be a comfort to readers.

Reply:
Done.